packages/fast-components-styles-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,106 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import {
+    applyFocusVisible,
+    applyLocalizedProperty,
+    Direction,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { ensureBrandNormal, hoverContrast } from "../utilities/colors";
+import outlinePattern from "../patterns/outline";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<PivotClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<PivotClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        pivot: {
+            position: "relative",
+            color: designSystem.foregroundColor,
+            overflow: "hidden",
+        },
+        pivot_tabList: {
+            display: "flex",
+            boxSizing: "border-box",
+        },
+        pivot_tab: {
+            minHeight: "32px",
+            padding: "0 12px",
+            fontSize: "14px",
+            whiteSpace: "nowrap",
+            display: "flex",
+            border: "2px solid",
+            borderColor: "transparent",
+            alignItems: "center",
+            boxSizing: "border-box",
+            userSelect: "none",
+            borderRadius: toPx(designSystem.cornerRadius),
+            "&:hover": {
+                opacity: "0.8",
+            },
+            "&:active": {
+                opacity: "0.6",
+            },
+            ...applyFocusVisible({
+                borderColor: designSystem.foregroundColor,
+            }),
+        },
+        pivot_tab__active: {},
+        pivot_activeIndicator: {
+            position: "absolute",
+            borderRadius: toPx(designSystem.cornerRadius),
+            top: "27px",
+            left: "-10px",

```

Comment:
LTR?

Reply:
this isn't effected by LTR is simply is for centering the sequence indicator